package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "next": "14.2.2",
+    "prettier": "^3.2.5",
```

Comment:
remove prettier. we have prettier in the dev dependencies. 

What is the difference between the dependencies and  the devDependencies?

Reply:
Gute Frage